app/models/manageiq/providers/vmware/infra_manager/event_catcher.rb

```diff
@@ -1,3 +1,5 @@
 class ManageIQ::Providers::Vmware::InfraManager::EventCatcher < ManageIQ::Providers::BaseManager::EventCatcher
   require_nested :Runner
+
+  self.rails_worker = !!worker_settings[:rails_worker]
```

Comment:
Will this cause access to the database at load time?

Reply:
I'm thinking no. this will not access the db.
Which surprised me. I had expected reading the settings would lookup the settings overrides.

```
>require "manageiq-providers-vmware"
# => false

> ManageIQ::Providers::Vmware::InfraManager::EventCatcher
# => ManageIQ::Providers::Vmware::InfraManager::EventCatcher

> ManageIQ::Providers::Vmware::InfraManager::EventCatcher.worker_settings
# => {:count=>1, :cpu_request_percent=>5, :cpu_threshold_percent=>100, :gc_interval=>900, :heartbeat_freq=>10, :heartbeat_timeout=>120, :memory_request=>524288000, :memory_threshold=>2147483648, :nice_delta=>1, :parent_time_threshold=>180, :poll=>1, :poll_escalate_max=>30, :poll_method=>"normal", :starting_timeout=>600, :stopping_timeout=>600, :flooding_events_per_minute=>30, :flooding_monitor_enabled=>true, :ems_event_page_size=>100, :ems_event_thread_shutdown_timeout=>10, :ems_event_max_wait=>60}

>Vm.first
PostgreSQLAdapter#log_after_checkout, connection_pool: size: 5, connections: 1, in use: 1, waiting_in_queue: 0
```

Reply:
Yeah we can always use a lambda if we need to